modules/react/src/components/export-video/constants.js

```diff
@@ -51,39 +51,45 @@ export const FORMATS = [
 export const RESOLUTIONS = [
   {
     value: '960x540',
-    label: 'Good 16:9 (540p)',
+    label: 'Good (540p)',
     width: 960,
-    height: 540
+    height: 540,
+    aspectRatio: '16:9'
```

Comment:
i would create a list of ratios to use:
```
export const RATIOS = ['4:3', '16:9']
```
or
```
export const RATIOS = {'4_3': '4:3', '16_9': '16:9'];
```
and re-use it

Reply:
Fixed